apps/entry/tasks.py

```diff
@@ -22,8 +26,9 @@ def __get_pdf_from_html(path, timeout=60, print_options={}):
     try:
         WebDriverWait(browser, timeout).until(lambda d: d.execute_script(
             'document.readyState == "complete"'))
-    except TimeoutException:
-        pass
+    except TimeoutException as e:
+        logger.error(f'Chromium timed out for {path}')
+        raise e
```

Comment:
This way will be better.
```suggestion
    except TimeoutException as e:
        logger.error(f'Chromium timed out for {path}', exc_info=True)
```